app/js/main.js

```diff
@@ -328,17 +333,55 @@ function startup() {
 
   // Preview a captured frame
   frameReelRow.addEventListener("click", function(e) {
-    if (e.target.className === "frame-reel-img") {
-      if (winMode !== "playback") {
-        switchMode("playback");
+    if (e.target.className.includes("frame-reel-img")) {
+      // For an unselected, non-keyframe
+      if (!e.target.className.includes("selected")) {
```

Comment:
Yea, you might want to do a find-replace for all instances of `classList.includes`...